<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoPointEditor</class>
 <widget class="QWidget" name="GeoPointEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>185</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Географическая координата</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="mapLabel">
     <property name="text">
      <string>Получить координату с карты</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="latLabel">
     <property name="text">
      <string>Широта:</string>
     </property>
     <property name="buddy">
      <cstring>latSpin</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lonLabel">
     <property name="text">
      <string>Долгота:</string>
     </property>
     <property name="buddy">
      <cstring>lonSpin</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="altLabel">
     <property name="text">
      <string>Высота:</string>
     </property>
     <property name="buddy">
      <cstring>lonSpin</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="altSpin">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Высота</string>
     </property>
     <property name="suffix">
      <string> м</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="latSpin">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Широта</string>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="lonSpin">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Долгота</string>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CoordEdit" name="lonEdit">
     <property name="maximum" stdset="0">
      <double>180.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CoordEdit" name="latEdit">
     <property name="maximum" stdset="0">
      <double>90.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="mapBtn">
     <property name="toolTip">
      <string>Указать координату на карте</string>
     </property>
     <property name="text">
      <string>M</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/meteo/icons/tools/center.png</normaloff>:/meteo/icons/tools/center.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="latCombo">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>с.ш.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ю.ш.</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="lonCombo">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>в.д.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>з.д.</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="searchCheck">
     <property name="text">
      <string>поиск станции</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CoordEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">meteo/commons/ui/custom/coordedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>latSpin</tabstop>
  <tabstop>latEdit</tabstop>
  <tabstop>latCombo</tabstop>
  <tabstop>lonSpin</tabstop>
  <tabstop>lonEdit</tabstop>
  <tabstop>altSpin</tabstop>
  <tabstop>mapBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
